logstash-core/lib/logstash/elasticsearch_client.rb

```diff
@@ -0,0 +1,94 @@
+# encoding: utf-8
+require "logstash/namespace"
+require "logstash/logging"
+require "elasticsearch"
+require "elasticsearch/transport/transport/http/manticore"
+        #
+        #     client = Elasticsearch::Client.new transport_class: Elasticsearch::Transport::Transport::HTTP::Manticore
```

Comment:
Can we remove this commented code line and add comments on what this class is actually doing?